addons/pos_restaurant/models/pos_restaurant.py

```diff
@@ -111,6 +111,7 @@ class RestaurantTable(models.Model):
     seats = fields.Integer('Seats', default=1, help="The default number of customer served at this table.")
     color = fields.Char('Color', help="The table's color, expressed as a valid 'background' CSS property value", default="#35D374")
     parent_id = fields.Many2one('restaurant.table', string='Parent Table', help="The parent table if this table is part of a group of tables")
+    child_ids = fields.One2many('restaurant.table', 'parent_id', string='Children Tables')
```

Comment:
No need to introduce this field. You can do `table["<-restaurant.table.parent_id"]` in JS to get the children tables.

Reply:
yes optionally we can use `get children()` method in place of new field